packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping-anchor.md

```diff
@@ -0,0 +1,197 @@
+
+_Content_
+
+- column title (icon and/or text)
+- 1 or more `nimble-mapping-icon`, `nimble-mapping-spinner`, or `nimble-mapping-empty` elements
+
+### Cell Template
+
+The cell template will essentially be a combination of the `TableColumnMappingCellView` and `TableColumnAnchorCellView`.
+
+```html
+<span class="reserve-icon-size"> ${x => x.visualizationTemplate} </span>
+When cellRecord.href present
+<nimble-anchor
+    href="${x => x.cellRecord.href}"
+    hreflang="${x => x.columnConfig.hreflang}"
+    ping="${x => x.columnConfig.ping}"
+    referrerpolicy="${x => x.columnConfig.referrerpolicy}"
+    rel="${x => x.columnConfig.rel}"
+    target="${x => x.columnConfig.target}"
+    type="${x => x.columnConfig.type}"
+    download="${x => x.columnConfig.download}"
+    underline-hidden
+    @mouseover="${(x, c) => setTitleWhenOverflow(...)}"
+    @mouseout="${(x, c) => removeTitle(...)}"
+>
+    ${cellState.cellRecord.label ?? cellState.cellRecord.href}
+</nimble-anchor>
+When cellRecord.href is missing
+<span
+    @mouseover="${(x, c) => setTitleWhenOverflow(...)}"
+    @mouseout="${(_x, c) => removeTitle(...)}"
+>
+    ${cellState.cellRecord.label}
+</span>
+```
+
+### Sorting
+
+Sorting will be based on the record value, following what is described in the [Mapping Table Column spec](./table-column-mapping.md#sorting), meaning the column will ultimately be sorted according to the icon "value" and not by the label or URL (as it is with the anchor column).
+
+### Sizing
+
+The column should support the same sizing modes as the text column, which is fractional widths plus minimum pixel widths.
+
+### Keyboard Interactions
+
+Arrowing to a anchor table cell should focus the link (if it is actually a link and not just a text span). Pressing `Enter` on a focused link will trigger navigation.
+
+### Accessibility Roles
+
+In the accessibility tree, the cells of an anchor column are instances of [`nimble-table-cell`](https://github.com/ni/nimble/blob/f663c38741e731bef91aa58e8fb2d1cec653b679/packages/nimble-components/src/table/components/cell/template.ts#L6) which has a `role` of [`cell`](https://w3c.github.io/aria/#cell). The cell then has a child `nimble-anchor`, which has a `role` of [`link`](https://w3c.github.io/aria/#link).
+
+### Angular integration
+
+Angular directives will be created for the column component. No component has form association, so a `ControlValueAccessor` will not be created.
+
+#### Angular Routing support
+
+Using the `TableColumnAnchor` in Angular requires the use of the [`NavigationGuard`](https://github.com/ni/nimble/blob/main/packages/angular-workspace/nimble-angular/table-column/anchor/nimble-table-column-anchor-navigation-guard.directive.ts) in order to allow proper routing within the Angular application.
```

Comment:
Should this say "Using the `TableColumnMappingAnchor`"? Or is this saying the `TableColumnAnchor` will somehow be reused?

Reply:
Think the intention was something along the lines of `TableColumnMappingAnchor` will follow the similar `NavigationGuard` directive pattern of `TableColumnAnchor`